sdk/servicebus/azure-servicebus/tests/mgmt_tests/test_mgmt_queues.py

```diff
@@ -302,6 +311,15 @@ def test_mgmt_queue_update_success(self, servicebus_namespace_connection_string,
             queue_description = mgmt_service.get_queue(queue_name)
             assert queue_description.lock_duration == datetime.timedelta(minutes=2)
 
+            # Update forwarding settings with entity name.
+            queue_description.forward_to = topic_name
+            queue_description.forward_dead_lettered_messages_to = topic_name
+            mgmt_service.update_queue(queue_description)
+
+            queue_description = mgmt_service.get_queue(queue_name)
+            assert queue_description.forward_dead_lettered_messages_to.endswith(".servicebus.windows.net/{}".format(topic_name))
+            assert queue_description.forward_to.endswith(".servicebus.windows.net/{}".format(topic_name))
```

Comment:
could we also test updating `forward_to` and `forward_dead_lettered_messages_to` to be None for the update operation?
```
            queue_description.forward_to = None
            queue_description.forward_dead_lettered_messages_to = None
            mgmt_service.update_queue(queue_description)

            queue_description = mgmt_service.get_queue(queue_name)
            assert not queue_description.forward_dead_lettered_messages_to
            assert not queue_description.forward_to
```

Reply:
meaning, setting it back to None after it has been set to a path?